src/Umbraco.Cms.Integrations.SEO.SemrushTools/Umbraco.Cms.Integrations.SEO.SemrushTools/Services/SemrushCachingService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+using Newtonsoft.Json;
+
+using Umbraco.Cms.Integrations.SEO.SemrushTools.Models.Dtos;
+using Umbraco.Core.Cache;
+
+namespace Umbraco.Cms.Integrations.SEO.SemrushTools.Services
+{
+    public class SemrushCachingService: ISemrushCachingService<RelatedPhrasesDto>
+    {
+        private readonly IAppPolicyCache _cache;
+
+        public SemrushCachingService(AppCaches appCaches)
+        {
+            _cache = appCaches.RuntimeCache;
+        }
+
+        public bool TryGetCachedItem(out RelatedPhrasesDto item, string key)
```

Comment:
Could re-order these parameters to match the usual order in these `Try...` methods, and as you have it for th one in `ISemrushService`.

Reply:
Item addressed.